elasticdl/python/common/data_reader.py

```diff
@@ -61,3 +64,45 @@ def create_shards(self):
             with closing(recordio.Index(p)) as rio:
                 f_records[p] = (start_ind, rio.num_records())
         return f_records
+
+
+class ODPSDataReader(AbstractDataReader):
+    def __init__(self, **kwargs):
+        AbstractDataReader.__init__(self, **kwargs)
+        self._kwargs = kwargs
```

Comment:
Please add a validating method here and print useful information to users if any of the following required arguments is missing:
+ project
+ access_id
+ access_key
+ endpoint
+ num_processes

Reply:
Done.